src/sentry/static/sentry/app/views/releases/list/shared/utils.jsx

```diff
@@ -1,16 +1,18 @@
-import {Client} from 'app/api';
+import qs from 'query-string';
 
 /**
- * Fetch organization releases given the query filters.
+ * Get query term for API given location.search
  *
- * @param {Object} organization
- * @param {Object} query
- * @returns {Promise<Array>}
+ * @param {String} search
+ * @returns {Object}
  */
-export function fetchOrganizationReleases(organization, query) {
-  const api = new Client();
 
-  const options = {query: {query: query.query}};
+export function getQuery(search) {
+  const query = qs.parse(search);
 
-  return api.requestPromise(`/organizations/${organization.slug}/releases/`, options);
+  return {
+    per_page: 50,
+    cursor: query.cursor,
```

Comment:
We may want to just extends `query` since you'll need more params from global selection.